UI/window-whats-new.hpp

```diff
@@ -0,0 +1,24 @@
+#pragma once
+
+#include <QPointer>
+#include <QDialog>
+#include <string>
+
+class QCefWidget;
+
+class OBSWhatsNew : public QDialog {
+	Q_OBJECT
+
+	QCefWidget *cefWidget = nullptr;
+
+public:
+	OBSWhatsNew(QWidget *parent, const std::string &url);
+	~OBSWhatsNew();
+
+	virtual void reject() override;
+	virtual void accept() override;
+};
+
+namespace {
```

Comment:
Move this into `window-basic-main.cpp` (probably towards the top before the first free-standing function).

It's the C++ variant of a C-style `static QPointer<OBSWhatsNew> obsWhatsNew;` declaration.